lib/engine/game/g_1844/entities.rb

```diff
@@ -99,7 +99,7 @@ module Entities
               {
                 type: 'choose_ability',
                 after_phase: '4',
-                when: 'owning_player_or_turn',
+                when: 'any',
```

Comment:
This change is unnecessary. Since you can only assign it to corporations you are a director of, limiting to their OR turns is sufficient. It reduces how much the ability has to show up without any play impact.

Reply:
Agree, I was mulling whether it was better to implement it 'as written' or for better playability